### ISSUE TITLE
Playlists and songs do NOT import in the right order, everything is inverted

### ISSUE THREAD

**Issue by Reflector92** [secuvera/SpotMyBackup#14]
I see that the playlists and songs DO NOT come in the right order they were when importing the .json file. Everything is just opposite, meaning my top playlist is at the bottom and continuing playlists. Also all songs are opposite, meaning the top song in a playlist will be at the bottom. 
How I fixed this was: Just import 2 times, when you import the inverted playlists, they will then be inverted again, so they come out the original order, and the songs too. Clever workaround, but this should be fixed too!

**Comment by sohambagchi** [comment]
Just sort each playlist by 'Recently Added'

**Comment by mr-varga** [comment]
I have the same issue, the script import the songs in reverse order, I mean first import the newest song  and then the old one. In this way if you connect shazam to spotify and and you shazam the new song, this is in opposite side in the playlist.

**Comment by FFY00** [comment]
Yes. The only thing imported in the right order is the starred songs.

**Comment by FFY00** [comment]
https://github.com/secuvera/SpotMyBackup/pull/16 this should fix it.
